internal/monitors/traefik/metadata.yaml

```diff
@@ -0,0 +1,280 @@
+    |-------------------|----------------|
+    | signalfx-agent    |     4.6.3+     |
+
+    ### INSTALLATION
+
+    ### CONFIGURATION
+    #### Traefik Configuration
+
+    Edit the Traefik configuration file, typically `traefik.toml`, to enable Traefik to expose prometheus metrics at an
+    endpoint. The endpoint is on path `/metrics` by default. When running the Traefik binary, the configuration file is
+    typically passed in as a command line argument. For example,
+
+    `./traefik -c traefik.toml`
+
+    However, when running the Traefik Docker image, the configuration file is mounted to volume
+    `/etc/traefik/traefik.toml`. For example,
+
+    `docker run -d -p 8080:8080 -p 80:80 -v $PWD/traefik.toml:/etc/traefik/traefik.toml`
+
+    If the Traefik configuration file is not available use the sample configuration file
+    <a target="_blank" href="https://raw.githubusercontent.com/containous/traefik/master/traefik.sample.toml">here</a>
+    to get started.
+
+    See <a target="_blank" href="https://docs.traefik.io/">here</a> for complete Traefik docs.
+
+    #### Smart Agent Configuration
+
+    The SignalFx Smart Agent must have network access to Traefik. This means for instance that, in a Docker environment,
+    the agent container needs to be in a shared network with the Traefik container.
+
+    Before we can send metrics to SignalFx, we need to make sure you are sending them to the correct SignalFx realm.
+    To determine what realm you are in (YOUR_SIGNALFX_REALM), check your profile page in the SignalFx web application
+    (click the avatar in the upper right and click My Profile). If you are not in the `us0` realm, you will need to set
+    the `SFX_INGEST_URL` configuration option to use the correct realm, as shown below.
+
+    You will also need to set the `SFX_ACCESS_TOKEN` configuration option to your SignalFx organization access token
+    (YOUR_SIGNALFX_API_TOKEN). For more information on authentication, see the API's
+    [Authentication documentation](https://developers.signalfx.com/basics/authentication.html).
+
+    The Docker run command to start the agent may look as follows:
+    ```
+    docker run --rm \
+    -e SFX_ACCESS_TOKEN=YOUR_SIGNALFX_API_TOKEN \
+    -e SFX_INGEST_URL=https://ingest.YOUR_SIGNALFX_REALM.signalfx.com \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v <agent.yaml path>:/etc/signalfx/agent.yaml \
+    --net <traefik container network> \
+    quay.io/signalfx/signalfx-agent:<version>
+    ```
+
+    Find and edit the SignalFx Smart Agent configuration file `agent.yaml` to enable the Docker observer and configure
+    this SignalFx Smart Agent traefik monitor. For example, the configuration below will cause the Smart Agent to query
+    the Docker Engine API for running containers with port 8080 exposed, attach dimension `metric_source=traefik` to any
+    Prometheus metrics at endpoint `<container ip>:8080/metrics` and export the metrics to SignalFx.
```

Comment:
Also don't mention `container_ip`.  This could just as well run with the `host` observer and it not be a container ip.